### PR TITLE
Feature - step 1 - done

### PR DESCRIPTION
**Pull request by olehvolynets**

### CODE REVIEW

File: spec/features/home_spec.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    visit root_path
+  end
+
+  it 'User can see latest books' do
+    within '#slider' do
+      expect(page).to have_selector('.item', count: 3, visible: false)
+    end
+  end
+
+  it 'User can see best sellers' do
```
Comment (6aBblKuH) [inline spec/features/home_spec.rb]:
```ruby
it "renders best sellers"
```

File: spec/features/home_spec.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    within '.best-sellers' do
+      expect(page).to have_selector('.best-seller-book', count: 4)
+    end
+  end
+
+  it 'User can see Get Started button' do
```
Comment (6aBblKuH) [inline spec/features/home_spec.rb]:
```ruby
it "renders Get Started button"
```

File: spec/features/home_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    get_started_button = find('button.btn.btn-default')
+    expect(get_started_button).to have_content('Get Started')
+  end
+end
```
Comment (6aBblKuH) [inline spec/features/home_spec.rb]: new line here
and as summary - u don`t need to check every DOM-element in new `it` block
u can group by them by some business logic
all elements tested in three different `it` blocks can be tested in one
if u had some troubles with rubocop-rspec - propose to update rubocop config

File: spec/models/book_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+# RSpec.describe Book, type: :model do
```
Comment (6aBblKuH) [inline spec/models/book_spec.rb]: ?

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -1,4 +1,7 @@
 require 'simplecov'
+SimpleCov.start do
+  minimum_coverage 95
+end
 
 RSpec.configure do |config|
   config.expect_with :rspec do |expectations|
```
Comment (6aBblKuH) [inline spec/spec_helper.rb]:
```ruby
config.order = :random
```
add this config please
randomizing tests provide better tests covering of ur application

File: db/migrate/20190928152408_create_books.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+class CreateBooks < ActiveRecord::Migration[5.2]
+  def change
+    create_table :books do |t|
+      t.string :title
+      t.string :authors
```
[the remark this answers is not in the document]
Comment (olehvolynets) [reply db/migrate/20190928152408_create_books.rb]: will be a relation when it comes to other models

File: .circleci/config.yml [hunk 7]
```diff
@@ -2,10 +2,13 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.6.3-stretch-node-browsers
+      - image: circleci/ruby:2.6.5-stretch-node-browsers
```
Comment (6aBblKuH) [inline .circleci/config.yml]: why do u use `stretch` image?

File: app/assets/javascripts/application.js [hunk 8]
```diff
@@ -12,6 +12,7 @@
 //
 //= require rails-ujs
 //= require activestorage
+//= require jquery
+//= require bootstrap-sass/assets/javascripts/bootstrap.js
```
Comment (6aBblKuH) [inline app/assets/javascripts/application.js]: should work even without extension `.js`

File: spec/controllers/pages_controller_spec.rb [hunk 9]
```diff
@@ -0,0 +1,3 @@
+# RSpec.describe PagesController, type: :controller do
```
Comment (6aBblKuH) [inline spec/controllers/pages_controller_spec.rb]: what about some specs here?
for example test for rendering needful view or successful status

File: spec/factories/books.rb [hunk 10]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :book do
+    title { 'MyString' }
```
Comment (6aBblKuH) [inline spec/factories/books.rb]: use FFaker please

File: spec/models/book_spec.rb (hunk 4 above)
Comment (6aBblKuH) [reply spec/models/book_spec.rb]: why u resolved conversation?
don`t leave specs commented if even subject does not provide any functionality

File: spec/controllers/pages_controller_spec.rb [hunk 11]
```diff
@@ -0,0 +1,15 @@
+RSpec.describe PagesController, type: :controller do
+  describe 'GET #home' do
+    before do
+      get :home
+    end
+
+    it 'has status 200' do
+      expect(response.status).to be 200
```
Comment (6aBblKuH) [inline spec/controllers/pages_controller_spec.rb]: same thing
`expect(response).to be_ok`

**Comment by 6aBblKuH** [comment]
U can fix last comment in next PR. Merge current PR by squashing strategy. Nice start 👍

**Pull request merged**